config/bootstrap.php

```diff
@@ -13,17 +13,23 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 
-// You can remove this if you are confident that your PHP version is sufficient.
+/**
```

Comment:
`/**` is docblock standard and these comments aren't for API docblocks so IMO it's better to use `/*` instead.